packages/ethers/src/account-signer.ts

```diff
@@ -102,6 +149,30 @@ export class AccountSigner extends Signer {
     );
   }
 
+  waitForUserOperationTransaction = async (
+    hash: `0x${string}`
+  ): Promise<`0x${string}`> => {
```

Comment:
oh sorry, can't you just do `this.provider.accountProvider.waitForUserOperationTransaction` here?

Reply:
yup! gonna bind it onto the signer class to mimic aa-alchemy and aa-core usage